src/types.ts

```diff
@@ -1269,6 +1277,11 @@ export interface RemoteConfig {
      */
     capturePerformance?: boolean | PerformanceCaptureConfig
 
+    /**
+     * Whether to capture History API events (pushState, replaceState, popstate)
+     */
+    captureHistoryEvents?: boolean
+
```

Comment:
do we need this on remote config? 
are you adding this in team settings as well? that's super cool!

Reply:
I was actually going to ask what the pattern is here; I see most of the `extensions` are on the remote config, but I don't think we would actually **need** it there, but if it is a good thing, sure!

Reply:
so, you only need to add it to remote config if you want to control whether it is active remotely from team settings

we don't control pageview capture iirc from remote settings so i think safe not to add this here